packages/twenty-server/src/database/typeorm/core/migrations/1724056827317-addInvitationTable.ts

```diff
@@ -0,0 +1,61 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddInvitationTable1724056827317 implements MigrationInterface {
```

Comment:
I don't understand why you'd need to create a dedicated table for this? Can't you just do everything with appToken? Having a table depending on appTokens doesn't bother me too much, but introducing foreign key in appTokens for a specific use-cases bother me more, as it's meant to be a generic table

Reply:
I misunderstood something. I thought you wanted to manage only the token through the appToken table. 
I will remove the invitation table, it will be easier to manage 👌